lib/system/web/src/lib.rs

```diff
@@ -328,3 +329,8 @@ impl NodeRemover for Node {
         self.remove_child(node).unwrap_or_else(panic_msg);
     }
 }
+
+#[wasm_bindgen(inline_js = "export function request_animation_frame2(f) { requestAnimationFrame(f) }")]
+extern "C" {
+    pub fn request_animation_frame2(closure: &Closure<dyn FnMut()>) -> i32;
```

Comment:
What is this for? I don't see any usages

Reply:
Right! That was a test, thanks for catching!